app/src/chainlit.py

```diff
@@ -15,17 +16,18 @@
 
 @cl.on_message
 async def main(message: cl.Message) -> None:
-    logger.info(f"Received: {message.content}")
+    logger.info(f"Received: {message.content!r}")
```

Comment:
TIL: https://stackoverflow.com/questions/44800801/in-python-format-f-string-strings-what-does-r-mean